test/mako/config/benchmark.go

```diff
@@ -38,13 +41,38 @@ func MustGetBenchmark() (*string, *string) {
 	return benchmarkKey, benchmarkName
 }
 
+type BenchmarkEnv struct {
```

Comment:
Why do you need separate dev+prod benchmark data?

It seems like you'd want to set the key/name based on dev/prod outside the container, so the container doesn't need to know about "dev" vs "prod" (so that adding "release" later, for example, requires no code changes).

Reply:
Currently, we have two benchmark for one perf test (dev and prod). When doing a perf test, it will install a configMap in the corresponding namespace to store the environment value (`dev` or `prod`) in configMap’s YAML file. 

Before my code change, this value is used in this function by `env, err := getEnvironment())` 
to check whether the container should publish data to dev benchmark or prod benchmark (by looking at benchmark_id from `env + “.config”`)

In the piece of code I added, I utilize this `env` variable to check whether I need to use `DEV_BENCHMARK_*` or `PROD_BENCHMARK_*` environment variable in the container.